EDDiscovery/UserControls/UserControlScanGrid.cs

```diff
@@ -195,22 +195,53 @@ void DrawSystem(HistoryEntry he, bool force)
                         bdDetails.Append("Volcanism".Tx(this) + ". ");
 
                     // have some ring?
-                    if (sn.ScanData.HasRings && sn.ScanData.IsStar == false)
+                    if (sn.ScanData.HasBelts || sn.ScanData.HasRings)
```

Comment:
HasBelts seems to be superfluous, and is not initialized until `DisplayString` is called.

Reply:
Yes, it's a placeholder for further improvements.